Tests/MSFT_xIPAddress.Tests.ps1

```diff
@@ -1,4 +1,17 @@
-﻿Remove-Module -Name MSFT_xIPAddress -Force -ErrorAction SilentlyContinue
+﻿if (! (Get-Module xDSCResourceDesigner))
+{
+    Import-Module -Name xDSCResourceDesigner
+}
+
+Describe 'Schema Validation MSFT_xIPAddress' {
+    Copy-Item -Path ((get-item .).parent.FullName) -Destination $(Join-Path -Path $env:ProgramFiles -ChildPath 'WindowsPowerShell\Modules\') -Force -Recurse
+    $result = Test-xDscResource MSFT_xIPAddress
+    It 'should pass Test-xDscResource' {
+        $result | Should Be $true
+    }
+}
+
+Get-Module MSFT_xIPAddress -All | Remove-Module -Force -ErrorAction:SilentlyContinue
 Import-Module -Name $PSScriptRoot\..\DSCResources\MSFT_xIPAddress -Force -DisableNameChecking
```

Comment:
Name checking should pass, If you didn't cause this issue, please file an issue and add comment mentioning the issue needs to be fixed.